system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -62,10 +62,16 @@ export class _TopSites extends React.PureComponent {
   }
 
   /**
-   * Return the TopSites to display based on prefs.
+   * Return the TopSites that are visible based on prefs and window width.
    */
-  _getTopSites() {
-    return this.props.TopSites.rows.slice(0, this.props.TopSitesRows * TOP_SITES_MAX_SITES_PER_ROW);
+  _getVisibleTopSites() {
+    // We hide 2 sites per row when not in the wide layout.
+    const WIDE_LAYOUT_BREAKPOINT = 1072; // 1056 + 16 for scrollbar
```

Comment:
The `1056` is seemingly arbitrary, so let's just have the comment be `// $break-point-widest from _variables.scss`

Reply:
And I suppose might as well just make the `1072` directly part of the `matchMedia` string arg.

Reply:
Interesting- I am seeing examples of people importing variables from .scss into js but I couldn't get it to work on first try so I'll punt on it for now.